ingestion/src/metadata/ingestion/source/pipeline/spline/metadata.py

```diff
@@ -171,6 +176,35 @@ def yield_pipeline_lineage_details(
             and lineage_details.executionPlan.inputs
             and lineage_details.executionPlan.output
         ):
+            target_to_sources_map = {}
+            for attr_id in lineage_details.executionPlan.extra.attributes:
```

Comment:
since execution plan is an optional field we might need to handle none here or within the model itself same goes for all the other fields like extra & attributes